google/provider.go

```diff
@@ -129,8 +129,8 @@ func Provider() terraform.ResourceProvider {
 			"google_organization_policy":                   resourceGoogleOrganizationPolicy(),
 			"google_project":                               resourceGoogleProject(),
 			"google_project_iam_policy":                    resourceGoogleProjectIamPolicy(),
-			"google_project_iam_binding":                   resourceGoogleProjectIamBinding(),
-			"google_project_iam_member":                    resourceGoogleProjectIamMember(),
+			"google_project_iam_binding":                   ResourceIamBinding(IamProjectSchema, NewProjectIamUpdater),
```

Comment:
s/Resource/resource?

Reply:
I wanted eventually to move the iam code into its own folder to have proper encapsulation. Only method starting with a capital letter would then be exported.

Reply:
But why is it different for binding and for member?

Reply:
My bad. Renamed so both start with a capital letter